docs/tutorials/03_emulation_sensitivity.ipynb

```diff
@@ -0,0 +1,516 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Scaling Sensitivity Analysis with Emulation"
```

Comment:
This could either say "scaling sensitivity analysis with autoemulate" or I'd leave just as is given the user is coming across this in the autoemulate docs

```suggestion
    "# Scaling Sensitivity Analysis"
```